dataeng/resources/vault-config.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+
+# make a directory to store vault token helper and config file
+mkdir ${WORKSPACE}/vault-config
+# get token helper from repo
+cp ${WORKSPACE}/jenkins-job-dsl/dataeng/resources/token-helper.py ${WORKSPACE}/vault-config/token-helper.py
+# remove the repo after getting token helper
+rm -rf ${WORKSPACE}/jenkins-job-dsl
+# create path for helper
+path_to_helper = ${WORKSPACE}"/vault-config/token-helper.py"
+# set up config for config file
+token_config = "$(echo token_helper = \"$path_to_helper\" )"
```

Comment:
I don't think you need to use command substitution here. I think it can be as simple as: `token_config="token_helper=$path_to_helper"`

Reply:
Actually I need to enclose path_to_helper in double quotes that why a substitution is required. If I do  token_config="token_helper=$path_to_helper" output will be 
token_helper = /vault-config/token-helper.py but I need output as 
token_helper = "/vault-config/token-helper.py"